resources/knative-eventing-kafka/templates/rbac.yaml

```diff
@@ -0,0 +1,180 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}-channel-controller
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "knative-kafka.labels" . | indent 4 }}
+
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-channel-controller
+  labels:
+{{ include "knative-kafka.labels" . | indent 4 }}
+rules:
+- apiGroups:
+  - "" # Core API Group
+  resources:
+  - services
+  verbs:
```

Comment:
Just a question: is not better readable to use the "oneline" form?
verbs: ["get", "post"]...

Reply:
imho that probably boils down to personal preference. In this case this is just a copy of what was already there.